examples/sysaudio/main.zig

```diff
@@ -61,6 +67,7 @@ pub fn update(app: *App, engine: *mach.Core) !void {
 // out using 1-log10(x*10) (google it to see how it looks, it's strong for most of the duration of
 // the note then fades out slowly.)
 pub const ToneEngine = struct {
+    sample_rate: f32 = 44100,
```

Comment:
This shouldn't be configured here / like this, it should come from the device/descriptor as it did before.

Reply:
The sample_rate set here is temporary. Admittedly this is not the best way to handle it. I'll pass the descriptor into the play function instead.